### ISSUE TITLE
error: command 'clang' failed with exit status 1

### ISSUE THREAD

**Issue by kingname** [flier/pyfasthash#29]
System is macOS High Sierra, 10.13.6
Python is 3.6.5

When I install pyhash, the blow error raised:

> clang -Wno-unused-result -Wsign-compare -Wunreachable-code -fno-common -dynamic -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.13.sdk/System/Library/Frameworks/Tk.framework/Versions/8.5/Headers -arch x86_64 -DT1HA0_AESNI_AVAILABLE=1 -DT1HA0_RUNTIME_SELECT=1 -DT1HA_USE_INDIRECT_FUNCTIONS=1 -c src/t1ha/src/t1ha0.c -o build/temp.macosx-10.13-x86_64-3.6/src/t1ha/src/t1ha0.o -msse4.2 -maes -mavx -mavx2
  <inline asm>:2:2: error: unknown directive
          .type   t1ha0, %gnu_indirect_function
          ^
  1 error generated.
  error: command 'clang' failed with exit status 1

**Comment by BooBSD** [comment]
I have the same problem!

**Comment by BooBSD** [comment]
I trying to install `pyhash` using the next command:

`pip install pyhash==0.9.1`

and got error:

```
    ranlib build/temp.macosx-10.13-x86_64-3.6/libsmhasher.a
    building 't1ha' library
    creating build/temp.macosx-10.13-x86_64-3.6/src/t1ha
    creating build/temp.macosx-10.13-x86_64-3.6/src/t1ha/src
    clang -Wno-unused-result -Wsign-compare -Wunreachable-code -fno-common -dynamic -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -DT1HA0_AESNI_AVAILABLE=1 -DT1HA0_RUNTIME_SELECT=1 -DT1HA_USE_INDIRECT_FUNCTIONS=1 -c src/t1ha/src/t1ha0.c -o build/temp.macosx-10.13-x86_64-3.6/src/t1ha/src/t1ha0.o -msse4.2 -maes -mavx -mavx2
    <inline asm>:2:2: error: unknown directive
            .type   t1ha0, %gnu_indirect_function
            ^
    1 error generated.
    error: command 'clang' failed with exit status 1
    
    ----------------------------------------
Command "/Users/artem/projects/venv/leapi-python/bin/python3.6 -u -c "import setuptools, tokenize;__file__='/private/var/folders/kr/n7bk9jg50tldc9_1sm9w539w0000gn/T/pip-install-aisa5q2v/pyhash/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /private/var/folders/kr/n7bk9jg50tldc9_1sm9w539w0000gn/T/pip-record-uwn55l1b/install-record.txt --single-version-externally-managed --compile --install-headers /Users/artem/projects/venv/leapi-python/bin/../include/site/python3.6/pyhash" failed with error code 1 in /private/var/folders/kr/n7bk9jg50tldc9_1sm9w539w0000gn/T/pip-install-aisa5q2v/pyhash/
```

My env:

`macOS-10.14.1`
`Xcode-10.1`

```
$ clang --version
Apple LLVM version 10.0.0 (clang-1000.11.45.5)
Target: x86_64-apple-darwin18.2.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
```

**Comment by flier** [comment]
it seems caused by `T1HA_USE_INDIRECT_FUNCTIONS`, just remove it

**Comment by kingname** [comment]
Thanks

**Comment by flier** [comment]
It should be fixed in 0.9.2, thanks
